deployment/default.yml.example

```diff
@@ -63,5 +63,7 @@ WorkerAutoScalingScheduleEndCapacity: '0'
 WorkerAutoScalingScheduleEndRecurrence: '0 1 * * *'
 ITSIBaseURL: ''
 ITSISecretKey: ''
+HydroShareBaseURL: ''
```

Comment:
We need to make sure that the corresponding values for this make it into the `default.yml` file on the file share. There is also a file on `civicci01` that is used by Jenkins for deployments (`/var/lib/jenkins/.aws/staging.yml`).

Reply:
Added to `default.yml` on fileshare and `civicci01`.